com/edocti/jintro/Exercitiul5.java

```diff
@@ -0,0 +1,137 @@
+import java.util.Scanner;
+
+public class Exercitiul5 {
+
+public static Scanner scan=new Scanner(System.in);
+
+
+
+private static boolean stringisEmpty(String input) {
+	return input==null || input.trim().length()==0;
+}
+
+//getAge
+private static int getAge(String line) {
+	if (stringisEmpty(line)) {
+		return -1; // string este gol
+	}
+    try {
+    	return Integer.parseInt(line.split(",")[2].trim());
+    } 
+    catch (NumberFormatException | ArrayIndexOutOfBoundsException e)
+    {
+    		    		return -1; }
+}
+
+//getName
+private static String getName(String line) {
+	if (stringisEmpty(line)) {
+		return "The String is empty"; // string este gol
+	}
+    try {
+	return line.split(",")[1].trim();
+    }
+    catch (NumberFormatException | ArrayIndexOutOfBoundsException e)
+    {
+    		    		return ""; }
+}
+
+
+//Display Names
+private static void displayNames (String lines) {
+	String[]line=lines.split("\n");
+	for (int i=0;i<=line.length-1;i++) {
+		System.out.println(getName(line[i]));
+	}
+}
+
+//Minimum Age
+private static void displayMinimumAge (String lines) {
+	String [] line=lines.split ("\n");
+	int min=getAge(line[0]);
+	int temp;
+		for (int i=0;i<line.length-1;i++) {
+		
+	    if (getAge(line[i])<getAge(line[i+1])) {
+		    temp=getAge(line[i]);
+		}
+	    else { temp=getAge(line[i+1]);
+	    }
+	    if (min>temp) {
+	    	  min=temp;
+	    	  }
+		}
+	System.out.println(min);
+}
+
+//Maximum Age
+private static void displayMaximumAge (String lines) {
+	String [] line=lines.split ("\n");
+	int max=getAge(line[0]);
+	int temp;
+		for (int i=0;i<line.length-1;i++) {
+		
+	    if (getAge(line[i])>getAge(line[i+1])) {
+		    temp=getAge(line[i]);
+		}
+	    else { temp=getAge(line[i+1]);
+	    }
+	    if (max<temp) {
+	    	  max=temp;
+	    	  }
+		}
+	System.out.println(max);
+}
+
+//MediumAge
+private static void displayMediumAge (String lines) {
+	String [] line=lines.split ("\n");

```

Comment:
no space between `String` and `[]`, but two spaces around `=`:
`String[] lines = ...`